search/make_catlas_minhashes.py

```diff
@@ -6,6 +6,7 @@
 import leveldb
 import shutil
 import json
+from collections import defaultdict
```

Comment:
where is this used?

Reply:
https://github.com/spacegraphcats/spacegraphcats/blob/b7e864d484d71f67540c98cc5f9bd572d5c89bda/search/make_catlas_minhashes.py#L136